src/client.js

```diff
@@ -0,0 +1,30 @@
+import Dashcore from '@dashevo/dashcore-lib'
+
+export default class Client {
+  constructor (socket) {
+    this.socket = socket
+  }
+
+  #modifiedTransaction (tx) {
+    return {
+      txId: tx.hash,
+      version: tx.version,
+      type: tx.type,
+      size: tx.toBuffer().length,
+      locktime: tx.nLockTime,
+      from: tx.inputs.map(input => input.script?.toAddress(Dashcore.Networks.testnet).toString()),
+      to: tx.outputs.map(output => {
+        return {
+          address: output.script?.toAddress(Dashcore.Networks.testnet).toString(),
+          valueSat: output.satoshis
+        }
+      })
+    }
+  }
+
+  broadcastMessage (message) {
```

Comment:
I don't think it is a broadcast, because it is linked with the specific client.
I would name it `sendMessage`